From 39ba76e57e4d67a22ee4a9500cc4a135c4da1bad Mon Sep 17 00:00:00 2001
From: David Gibson <david@gibson.dropbear.id.au>
Date: Fri, 10 Aug 2018 13:55:38 +1000
Subject: [PATCH 6/9] pylibfdt: Don't silence setup.py when V=1

At the moment we unconditionally pass --quiet to setup.py.  Change that to
get more debugging output from it when V=1 is passed to make.

Signed-off-by: David Gibson <david@gibson.dropbear.id.au>
Signed-off-by: Simon Glass <sjg@chromium.org>
---

 pylibfdt/Makefile.pylibfdt | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/pylibfdt/Makefile.pylibfdt b/pylibfdt/Makefile.pylibfdt
index bbd6518..005cb25 100644
--- a/pylibfdt/Makefile.pylibfdt
+++ b/pylibfdt/Makefile.pylibfdt
@@ -5,10 +5,15 @@ PYLIBFDT_srcs = $(PYLIBFDT_srcdir)/libfdt.i
 PYMODULE = $(PYLIBFDT_objdir)/_libfdt.so
 
 SETUP = $(PYLIBFDT_srcdir)/setup.py
+SETUPFLAGS =
+
+ifndef V
+SETUPFLAGS += --quiet
+endif
 
 define run_setup
 	VERSION="$(dtc_version)"
-	$(SETUP) --quiet $(1)
+	$(SETUP) $(SETUPFLAGS) $(1)
 endef
 
 $(PYMODULE): $(PYLIBFDT_srcs) $(LIBFDT_archive) $(SETUP)
-- 
2.19.0.444.g18242da7ef-goog

